docs/specs/README.md

```diff
@@ -0,0 +1,28 @@
+# Pipeline specifications
+
+This folder contains pipeline specification documents. Each document
+provides a written specification of a pipeline, intended as a basis
+for implementation.
+
+## Versioning policy
+
+Each specification will include a three-part version number. The
+version number will be incremented as follows:
+
+* Micro version bump - Add or clarify information in spec, but no
```

Comment:
When we recently implemented semver we only did major and minor versions. We haven't actually changed to adding patch or micro versions back in, but it's come up a few times as something that would feel useful. I would include changes to memory/disk and comments as micro changes as well.

Thoughts?

Reply:
That sounds like a good policy for versioning pipeline implementations. 

But just to be clear, the README here is describing how to version the pipeline specification documents.

I.e., we will have a pipeline spec, and one or more implementations of that spec. The spec and the implementation will each need their own version numbers. And we'll probably need some way of capturing the fact that a given implementation version will be based on a given spec version.